extensions/elasticsearch-rest-client-common/deployment/src/main/java/io/quarkus/elasticsearch/restclient/common/deployment/ElasticsearchRestClientProcessor.java

```diff
@@ -14,10 +13,4 @@ public void build(BuildProducer<ExtensionSslNativeSupportBuildItem> extensionSsl
         extensionSslNativeSupport.produce(new ExtensionSslNativeSupportBuildItem(Feature.ELASTICSEARCH_REST_CLIENT_COMMON));
     }
 
-    @BuildStep
-    public ReflectiveClassBuildItem registerForReflection() {
-        return new ReflectiveClassBuildItem(true, true,
-                "org.apache.logging.log4j.message.ReusableMessageFactory",
-                "org.apache.logging.log4j.message.DefaultFlowMessageFactory");
-    }
```

Comment:
You moved this from the low-level client module to the high-level client module. I don't understand why.

As far as I can tell, the low-level client uses Log4j too. And it can be used without the high-level client.

So... why?

Reply:
Wait, sorry, you're saying in your comment that the low-level client doesn't depend on log4j-api. It's weird, I would have sworn it did... let me check...

Reply:
You are right. This can be ignored, sorry about the noise.